docs/markdown/android/textarea.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Text area
```

Comment:
I think this should be `TextArea` without the space

Reply:
I adjusted both Text Field and Search Field to match as they also included a space.